symfony/plugins/orangehrmAdminPlugin/Controller/BaseAdminController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Admin\Controller;
+
+use OrangeHRM\Core\Authorization\Controller\CapableViewController;
+use OrangeHRM\Core\Controller\AbstractVueController;
+use OrangeHRM\Core\Helper\VueControllerHelper;
+use OrangeHRM\Core\Traits\UserRoleManagerTrait;
+use OrangeHRM\Framework\Http\Request;
+
+abstract class BaseAdminController extends AbstractVueController implements CapableViewController
```

Comment:
Actually, I had the need to implement this controller since I fetched the data group permissions to the UI level. Even though admin screens are not related to an Employee entity, I thought we might need to get data group permissions to control UI elements. If I remove this now, I might have to remove the places where I use permissions in Vue controllers too. Do I need to proceed removing this?